plugins/scaffolder-backend/src/scaffolder/actions/builtin/ci/githubActionsDispatch.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { InputError } from '@backstage/errors';
+import {
+  GithubCredentialsProvider,
+  ScmIntegrationRegistry,
+} from '@backstage/integration';
+import { Octokit } from '@octokit/rest';
+import { createTemplateAction } from '../../createTemplateAction';
+import { Config } from '@backstage/config';
+
+const host = 'github.com';
```

Comment:
We weren't sure what was the best approach to pass the host. We started off by having `repoUrl` as a parameter but then decided to split it into `owner` and `repoName`, without realising that this might not work with GitHub Enteprise.

We don't necessarily have experience with GHE so we weren't 100% sure if this is not going to work.

What would you recommend?